text/0035-conviction-voting-delegation-modifications

```diff
@@ -0,0 +1,77 @@
+3. Make a change so that when a delegate votes abstain their delegated votes also vote abstain.
+4. Allow a Delegator to delegate/ undelegate their votes for all tracks with a single call. 
+
+## Motivation
+
+It has become clear since the launch of OpenGov that there are a few common tropes which pop up time and time again:
+1. The frequency of referenda is often too high for network participants to have sufficient time to review, comprehend, and ultimately vote on each individual referendum. This means that these network participants end up being inactive in on-chain governance.
+2. There are active network participants who are reviewing every referendum and are providing feedback in an attempt to help make the network thrive – but often time these participants do not control enough voting power to influence the network with their positive efforts.
+3. Delegating votes for all tracks currently requires long batched calls which result in high fees for the Delegator - resulting in a reluctance from many to delegate their votes.
+
+We believe (based on feedback from token holders with a larger stake in the network) that if there were some changes made to delegation mechanics, these larger stake holders would be more likely to delegate their voting power to active network participants – thus greatly increasing the support turnout.
+
+## Stakeholders
+
+The primary stakeholders of this RFC are:
+
+- The Polkadot Technical Fellowship who will have to research and implement the technical aspects of this RFC
+- DOT token holders in general 
+
+## Explanation
+
+This RFC proposes to make 4 changes to the convictionVoting pallet logic in order to improve the user experience of those delegating their voting power to another account. 
+
+1. **Allow a Delegator to vote independently of their Delegate if they so desire** – this would empower network participants to more actively delegate their voting power to active voters, removing the tedious steps of having to undelegate across an entire track every time they do not agree with their delegate's voting direction for a particular referendum.
+
+2. **Allow nested delegations – for example Charlie delegates to Bob who delegates to Alice – when Alice votes then both Bob and Charlie vote alongside Alice (in the current runtime Charlie will not vote when Alice votes)** – This would allow network participants who control multiple (possibly derived) accounts to be able to delegate all of their voting power to a single account under their control, which would in turn delegate to a more active voting participant. Then if the delegator wishes to vote independently of their delegate they can control all of their voting power from a single account, which again removes the pain point of having to issue multiple undelegate extrinsics in the event that they disagree with their delegate.
```

Comment:
AFAIK nested delegation (or at least some simple primitive of it) is already implemented.

Am I wrong?

Reply:
Nested delegations do not work on Kusama or Polkadot, there may be some code somewhere that hasn't yet been implemented that I am not aware of.